docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -186,6 +186,23 @@ Property | Description
 `node`  | The current DOM node.
 `idx` | Index of the current node among other nodes in the matching set.
 
+```js
+Selector('ul').filter((node, idx) => {
+    // node === a <ul> node
+    // idx === index of the current <ul> node
+}));
+```
+
+The `dependencies` parameter allows you to pass objects to the `filterFn` function's scope where they appear as variables.
```

Comment:
I'm not sure if such a connection would be obvious.

Also, in a ClientFunction example they will see an options object

```
{ dependencies: { fn } }
```

While in selector's filter it's just dependencies

```
{ fn }
```

It can be misleading.